src/shared/containers/SubmissionPage.jsx

```diff
@@ -64,7 +69,8 @@ class SubmissionsPageContainer extends React.Component {
       challenge,
       track,
     } = this.props;
-
+    // When the user is waiting for their submission to upload, the survey should appear
+    Sprig('track', 'onUploadSubmission');
```

Comment:
If you want to use the sdk, you can do `sprig.track('onUploadSubmission')`.

You might also consider adding the user info to the events.

https://github.com/topcoder-platform/platform-ui/blob/TCA-790_assessments/src-ts/lib/functions/survey-functions/survey.functions.ts